tests/service/Symbols.fs

```diff
@@ -170,6 +170,26 @@ let x = 123
         |> Option.map (fun su -> su.Symbol :?> FSharpMemberOrFunctionOrValue)
         |> Option.iter (fun symbol -> symbol.Attributes.Count |> shouldEqual 1)
 
+    [<Test>]
+    let ``Test obsolete inside obsolete`` () =
+        let source = """
+open System
+
+[<Obsolete>]
+let add a b = a + b
+
+[<Obsolete>]
+let newAdd a b = add a b
+"""
+
+        let fileName, options = mkTestFileAndOptions source [| "--noconditionalerasure" |]
+        let _, checkResults = parseAndCheckFile fileName source options
+
+        checkResults.Diagnostics
+        |> Array.exists (fun diag -> diag.ErrorNumber = 44 (* this construct is deprecated *))
```

Comment:
Better to use an `Assert.NotEmpty`

Reply:
Assert.IsEmpty does not take a filter argument. If you mean to add filter before - what's the point?

Reply:
You should test for no warnings emitted, no filter

Reply:
Why? I'm only testing that there's no 44 warning.

Reply:
And you allow other warnings to be emitted?

Reply:
Yes. There's in fact an error because it doesn't compile without module and/or entry point or something like this. I'm not sure how I'm supposed to address it. *But* there are far more important things currently - for example, I'm trying to understand how to traverse the enclosing entity tree to implement the feature. Some nit fixes in a test is the last thing I'm concerned about, currently